Build via Github action fails although code did not change

The following build job failed, although there have not been any code changes:
https://github.com/coatrack/coatrack/actions/runs/3832354857/jobs/6522560746

The error message is:

```
Warning:  Error injecting: org.springframework.boot.maven.RepackageMojo
java.lang.TypeNotPresentException: Type org.springframework.boot.maven.RepackageMojo not present
    at org.eclipse.sisu.space.URLClassSpace.loadClass (URLClassSpace.java:147)
    at org.eclipse.sisu.space.NamedClass.load (NamedClass.java:46)
    at org.eclipse.sisu.space.AbstractDeferredClass.get (AbstractDeferredClass.java:48)
[ . . . ]
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.lang.UnsupportedClassVersionError: org/springframework/boot/maven/RepackageMojo has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
    at java.lang.ClassLoader.defineClass1 (Native Method)
    at java.lang.ClassLoader.defineClass (ClassLoader.java:1017)
    at java.security.SecureClassLoader.defineClass (SecureClassLoader.java:174)
[ . . . ]
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for CoatRack 2.0.0-SNAPSHOT:
[INFO] 
[INFO] CoatRack ........................................... SUCCESS [  4.458 s]
[INFO] coatrack-api ....................................... SUCCESS [  4.825 s]
[INFO] coatrack-spring-boot-parent ........................ FAILURE [  1.684 s]
[INFO] coatrack-proxy ..................................... SKIPPED
[INFO] coatrack-admin ..................................... SKIPPED
[INFO] coatrack-config-server ............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  17.423 s
[INFO] Finished at: 2023-01-03T19:21:57Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal org.springframework.boot:spring-boot-maven-plugin:3.0.1:repackage (default) on project coatrack-spring-boot-parent: Execution default of goal org.springframework.boot:spring-boot-maven-plugin:3.0.1:repackage failed: Unable to load the mojo 'repackage' in the plugin 'org.springframework.boot:spring-boot-maven-plugin:3.0.1' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: org/springframework/boot/maven/RepackageMojo has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
Error:  -----------------------------------------------------
Error:  realm =    plugin>org.springframework.boot:spring-boot-maven-plugin:3.0.1
Error:  strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
Error:  urls[0] = file:/home/runner/.m2/repository/org/springframework/boot/spring-boot-maven-plugin/3.0.1/spring-boot-maven-plugin-3.0.1.jar
[ . . . ]
```

--- a/spring-boot/pom.xml
+++ b/spring-boot/pom.xml
@@ -143,6 +143,7 @@
                 <plugin>
                     <groupId>org.springframework.boot</groupId>
                     <artifactId>spring-boot-maven-plugin</artifactId>
+					<version>2.7.7</version>
                     <executions>
                         <execution>
                             <goals>

--- a/spring-boot/pom.xml
+++ b/spring-boot/pom.xml
@@ -143,7 +143,7 @@
                 <plugin>
                     <groupId>org.springframework.boot</groupId>
                     <artifactId>spring-boot-maven-plugin</artifactId>
-					<version>2.7.7</version>
+                    <version>2.7.7</version>
                     <executions>
                         <execution>
                             <goals>

